src/BingoGame.Indexer.CA/Entities/Guardian.cs

```diff
@@ -0,0 +1,15 @@
+using Nest;
+
+namespace BingoGame.Indexer.CA.Entities;
+
+public class Guardian
```

Comment:
It's not used anywhere. We should delete it.

Reply:
noted, deleted